### PR TITLE
chore(demo): add sanitizer info to icons page

### LINKED ISSUE #454: [DOCS] Add sanitizer info to icons page
When customizing icons sanitizer knowledge is important

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #454 

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/editor/editor.template.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <tui-doc-page header="Editor" package="ADDON-EDITOR" type="components">
     <ng-template pageTab>
-        <p i18n>Rich Text Editor for using with Angular forms</p>
+        <p i18n>
+            Rich Text Editor for using with Angular forms. For safety reasons
+            use a <a tuiLink href="/icon-set#sanitizer">sanitizer</a> with this
```
Comment (MarsiBarsi) [inline projects/demo/src/modules/components/editor/editor.template.html:5]: routerLink

File: projects/demo/src/modules/customization/icon-set/icon-set.template.html [hunk 2]
```diff
@@ -50,6 +50,29 @@
...
+            If you use unsafe icons or have kind of WYSISYG editor in your app
+            you need to provide TUI_SANITIZER token, which is responsible for
+            removing malicious code from the svg. We recommend to use
+            <a
+                tuiLink
+                href="https://github.com/TinkoffCreditSystems/ng-dompurify"
+                target="_blank"
+                >NgDompurify</a
+            >
+            as sanitizer. This library implements DOMPurify as Angular Sanitizer
```
Comment (MarsiBarsi) [inline projects/demo/src/modules/customization/icon-set/icon-set.template.html:65]: nit: as a sanitizer

File: projects/demo/src/modules/customization/icon-set/icon-set.template.html [hunk 3]
```diff
@@ -50,6 +50,29 @@
         folder and provide <code>TUI_ICONS_PATH</code> as seen in example below:
     </p>
 
+    <section id="sanitizer">
+        <h2>Sanitizer</h2>
+        <p i18n>
+            If you use unsafe icons or have kind of WYSISYG editor in your app
+            you need to provide TUI_SANITIZER token, which is responsible for
```
Comment (MarsiBarsi) [inline projects/demo/src/modules/customization/icon-set/icon-set.template.html:57]: <code>TUI_SANITIZER</code>

File: projects/demo/src/modules/customization/icon-set/icon-set.template.html (hunk 2 above)
Comment (MarsiBarsi) [reply projects/demo/src/modules/customization/icon-set/icon-set.template.html:65]: let's make DOMPurify a link to the project

**Review by MarsiBarsi — CHANGES_REQUESTED** [review]

File: projects/demo/src/modules/customization/icon-set/icon-set.template.html [hunk 4]
```diff
@@ -50,6 +50,36 @@
         folder and provide <code>TUI_ICONS_PATH</code> as seen in example below:
     </p>
 
+    <section id="sanitizer">
+        <h2>Sanitizer</h2>
+        <p i18n>
+            If you use unsafe icons or have kind of WYSISYG editor in your app
```
Comment (waterplea) [inline projects/demo/src/modules/customization/icon-set/icon-set.template.html:56]: > or Editor component (with a link to it)

**Review by waterplea — APPROVED** [review]
Small comment, other than that LGTM

**Review by MarsiBarsi — APPROVED** [review]

**Pull request merged**